src/state/apis/zapper/validators.ts

```diff
@@ -467,12 +445,7 @@ export type V2AppsBalancesResponseType = Infer<typeof V2AppsBalancesResponse>
 
 const V2AppTokenResponse = z.object({
   address: z.string(),
-  network: z.enum(SupportedZapperNetworkIncludeUnsupported),
-})
-
-const V2AppSupportedNetworkResponse = z.object({
-  network: z.enum(SupportedZapperNetworkIncludeUnsupported),
-  actions: z.array(z.string()),
+  network: z.string(),
```

Comment:
This'll do and will avoid us having to update the schema everytime zapper supports new networks - we have `ZAPPER_NETWORKS_TO_CHAIN_ID_MAP` and `CHAIN_ID_TO_ZAPPER_NETWORK_MAP` mappings either way